dali/python/backend_impl.cc

```diff
@@ -1322,6 +1275,35 @@ py::dict ExecutorMetaToDict(const ExecutorMetaMap &meta) {
   return d;
 }
 
+template <typename Backend>
+void ExposeEagerOperator(py::module &m, const char *name) {
+  py::class_<EagerOperator<Backend>>(m, name)
+      .def(py::init([](const OpSpec &op_spec) {
+             return std::make_unique<EagerOperator<Backend>>(op_spec);
+           }),
+           "op_spec"_a)
+      .def("__call__",
+           [](EagerOperator<Backend> &op,
+              const std::vector<
+                  std::shared_ptr<TensorList<typename Backend2Types<Backend>::InBackend>>> &inputs,
+              const std::unordered_map<std::string, std::shared_ptr<TensorList<CPUBackend>>>
+                  &kwargs) { return op.Run(inputs, kwargs); })
+      .def("reader_meta",
+           [](EagerOperator<Backend> &op) { return ReaderMetaToDict(op.GetReaderMeta()); });
+}
+
+void ExposePipelineDebug(py::module &m) {
+  py::class_<PipelineDebug>(m, "PipelineDebug")
+      .def(py::init([](int batch_size, int num_threads, int device_id, bool set_affinity = false) {
```

Comment:
Question/nitpick: Is there a reason not to name the args (I mean passing the`"batch_size"_a` etc.)